recipes-core/system-api/system-api_v0.3.0.bb

```diff
@@ -9,9 +9,8 @@ INITSCRIPT_NAME = "system-api-init"
 INITSCRIPT_PARAMS = "defaults 81"
 
 GO_IMPORT = "github.com/flashbots/system-api"
-SRC_URI = "git://${GO_IMPORT};protocol=https;branch=main \
+SRC_URI = "git://${GO_IMPORT};protocol=https;nobranch=1;rev=v0.3.0 \
            file://system-api-init"
-SRCREV = "v0.3.0"
```

Comment:
Ah I see that the [tag](https://github.com/flashbots/system-api/tree/v0.3.0) is not based on any branch or on a fork/different deleted branch maybe.
We should probably fix the tag issue there instead of changing this here because this should  be the default way to fetch a repo and tag/commit.

Reply:
@metachris Can you please create a proper tag in the system api repo that is based on main, not a dangling commit?

Reply:
created a v0.4.0 tag